src/patternfly/components/Touchspin/examples/Touchspin.md

```diff
@@ -0,0 +1,275 @@
+---
+id: Touchspin
+section: components
+cssPrefix: pf-c-touchspin
+beta: true
+---
+
+## Examples
+### Default
+```hbs
+{{#> touchspin}}
+  {{#> touchspin-main}}
+    {{#> input-group}}
+      {{#> touchspin-control}}
+        {{#> button button--modifier="pf-m-control" button--attribute='id="textAreaButton1"'}}
+          {{#> touchspin-icon}}
+            <i class="fas fa-minus" aria-hidden="true"></i>
+          {{/touchspin-icon}}
+        {{/button}}
+      {{/touchspin-control}}
+      {{#> form-control controlType="input" input="true" form-control--attribute='type="text" value="90" id="textInput1" name="textInput1" aria-label="Input touchspin"'}}
+      {{/form-control}}
+      {{#> touchspin-control}}
+        {{#> button button--modifier="pf-m-control"}}
+          {{#> touchspin-icon}}
+            <i class="fas fa-plus" aria-hidden="true"></i>
+          {{/touchspin-icon}}
+        {{/button}}
+      {{/touchspin-control}}
+    {{/input-group}}
+  {{/touchspin-main}}
+{{/touchspin}}
+```
+
+### Active touchspin
+```hbs
+{{#> touchspin}}
+  {{#> touchspin-main touchspin-main--modifier="pf-m-active"}}
+    {{#> input-group}}
+      {{#> touchspin-control}}
+        {{#> button button--modifier="pf-m-control" button--attribute='id="textAreaButton1"'}}
+          {{#> touchspin-icon}}
+            <i class="fas fa-minus" aria-hidden="true"></i>
+          {{/touchspin-icon}}
+        {{/button}}
+      {{/touchspin-control}}
+      {{#> form-control controlType="input" input="true" form-control--attribute='type="text" value="91" id="textInput2" name="textInput2" aria-label="Input touchspin"'}}
+      {{/form-control}}
+      {{#> touchspin-control}}
+        {{#> button button--modifier="pf-m-control pf-m-expanded"}}
```

Comment:
From looking at the mockups, I don't think we need expanded on the form control or buttons. That styling would just come from hover/focus states. I think that would be the most predictable since that's how the button/form controls work in other contexts.

Reply:
Okay that sounds good to me